src/main/js/util/fieldMapping.js

```diff
@@ -13,18 +13,21 @@ import DescriptorOption from 'component/common/DescriptorOption';
 
 function trimValue(items) {
     const { value } = items;
-    const trimmedValue = (Array.isArray(value)) ? value[0] : value;
+    return (Array.isArray(value)) ? value[0] : value;
```

Comment:
What is the point of this? The value is never trimmed, it just returns the first value if its an array? Otherwise return the value?

Reply:
The name trim can probably be changed, but the intent of this function is to handle the arrays we use on the backend. All our fields expect a certain type of data to be passed to it, but the backend always returns an array of strings. If a field uses this function, it will confirm we have an array of strings then grab the first one (assuming there's only one item). If it's not an array, then it's assumed we have the exact value we need and we return it.

Reply:
I think a name change would fix my confusion here. Maybe, extractSingleValue?